docs/faq.rst

```diff
@@ -167,7 +167,141 @@ The maximum parallelism depends on the number of partitions.
          against a two-partition input topic, only two servers perform the actual work, but the other eight will run an “idle”
          query.
 
+=================================================================================
+Can KSQL connect to an Apache Kafka cluster over SSL and authenticate using SASL?
+=================================================================================
+
+Yes. Internally, KSQL uses standard Kafka consumers and producers.
+The procedure to securely connect KSQL to Kafka is the same as connecting any app to Kafka.
+
+For example, you can add the following entries to the KSQL server configuration file
+(ksqlserver.properties). This configuration enables KSQL to connect to a secure Kafka
+cluster using _PLAIN_ SASL (other options include GSSAPI / Kerberos), where the SSL
+certificates have been signed by a CA trusted by the default JVM trust store:
+
+.. code:: bash
+    security.protocol=SASL_SSL
+    sasl.mechanism=PLAIN
+    sasl.jaas.config=\
+        org.apache.kafka.common.security.plain.PlainLoginModule required `
+        username="<name of the user KSQL should use>" `
+        password="<the password>";
+
+The exact settings you will need will vary depending on what SASL mechanism your
+Kafka cluster is using and how your SSL certificates are signed. For full details,
+please refer to the `Security section of the Kafka documentation
+<http://kafka.apache.org/documentation.html#security>`__.
 
+====================================
+Will KSQL work with Confluent Cloud?
+====================================
+
+Running KSQL against an Apache Kafka cluster running in the cloud is pretty straight forward.
+To do so, add the following to the KSQL configuration file, (ksql-server.properties):
+
+... code:: bash
+    bootstrap.servers=<a comma separated list of the the ccloud broker endpoints. eg. r0.great-app.confluent.aws.prod.cloud:9092,r1.great-app.confluent.aws.prod.cloud:9093,r2.great-app.confluent.aws.prod.cloud:9094>
+    ksql.sink.replicas=3
+    replication.factor=3
+    security.protocol=SASL_SSL
+    sasl.mechanism=PLAIN
+    sasl.jaas.config=\
+        org.apache.kafka.common.security.plain.PlainLoginModule required \
+        username="<confluent cloud access key>" \
+        password="<confluent cloud secret>";
+
```

Comment:
add pointer to the new CCloud topic, e.g. `:ref:`install_ksql-ccloud``. see https://github.com/confluentinc/ksql/pull/955/files